Automattic/woocommerce-payments
# Add E2E tests for shopper multi-currency checkout

Closes #7350 

#### Changes proposed in this Pull Request
- Implement E2E tests for multi-currency checkout functionality.

**Test cases:**
1. The order received page displays the correct currency when checking out in `EUR` and `USD`.
2. My Account order details table shows the correct currency.
3. My Account single order detail page reflects the correct currency.

#### Testing instructions

1. Set up a clean E2E local environment.
2. Execute `npm run test:e2e-dev shopper-checkout-multi-currency.spec.js`.
3. Confirm that the tests pass in your local environment.
4. Verify that all E2E tests pass in the GitHub (GH) checks.

-------------------


**Post merge**
